.golangci.yml

```diff
@@ -36,7 +35,6 @@ linters:
     - staticcheck
     - unconvert
     - unused
-    - varcheck
```

Comment:
Why did you disabled it?

Reply:
Because it is deprecated: https://golangci-lint.run/usage/linters/